dtm0/ut/main.c

```diff
@@ -29,6 +29,7 @@
 #include "ut/ut.h"
 #include "cas/cas.h"
 #include "cas/cas_xc.h"
+#include "dtm0/service.h"             /* m0_dtm0_service_find */
```

Comment:
It looks like this thing has already been included

Reply:
Removed it.